hello.go

```diff
@@ -8,6 +8,6 @@ func main() {
 
 func sayHello() {
     fmt.Println("Hello world!!! circle ci - api token deleted")
-//    return
-//    fmt.Println("Unreachable line") // got vet error test
+    return
+    fmt.Println("Unreachable line") // got vet error test
```

Comment:
**[govet]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
unreachable code

Reply:
ok i will fix this.